screenshot.css

```
@@ -0,0 +1,12 @@
+iframe,
+.avatar__photo,
+img[src$='.gif'],
+.DocSearch-Button-Keys > kbd,
+[class*='playgroundPreview'] {
+  visibility: hidden;
+}
+
+.theme-last-updated,
+.docusaurus-mermaid-container {
+  display: none;
+}

```

Please implement the recommendation from the comment in the issue (for video, for example)

```suggestion
/* Iframes can load lazily */
iframe,
/* Avatars can be flaky due to using external sources: GitHub/Unavatar */
.avatar__photo,
/* Gifs load lazily and are animated */
img[src$='.gif'],
/* Algolia keyboard shortcuts appear with a little delay */
.DocSearch-Button-Keys>kbd,
/* The live playground preview can often display dates/counters */
[class*='playgroundPreview'] {
  visibility: hidden;
}

/* Different docs last-update dates can alter layout */
.theme-last-updated,
/* Mermaid diagrams are rendered client-side and produce layout shifts */
.docusaurus-mermaid-container {
  display: none;
}

/* Videos are laggy and the loading spinner is shown as change on Argos */
video {
  visibility: hidden;
}
```